diff --git a/CMakeLists.txt b/CMakeLists.txt
index c20e3abd1..c4bd7eeef 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -187,8 +187,10 @@ if(NOT GCG_DEV_BUILD)
    if(SCIP_FOUND)
       # SCIP headers can be directly included
       include_directories(${SCIP_INCLUDE_DIRS})
-
+      
+      message("hier0 hier0 hier0 hier0 hier0 hier0 hier0 hier0 hier0 hier0 hier0 hier0 hier0 hier0 hier0 hier0 hier0")
       if(ZLIB AND NOT TARGET ZLIB::ZLIB)
+         message("hier hier hier hier hier hier hier hier hier hier hier hier hier hier hier hier hier")
          find_package(ZLIB REQUIRED)
       endif()
       if(READLINE)
